src/test/java/org/openrewrite/java/testing/mockito/MockitoWhenOnStaticToMockStaticTest.java

```diff
@@ -21,13 +21,24 @@
 import org.openrewrite.java.JavaParser;
 import org.openrewrite.test.RecipeSpec;
 import org.openrewrite.test.RewriteTest;
-import org.openrewrite.test.SourceSpec;
+import org.openrewrite.test.SourceSpecs;
 import org.openrewrite.test.TypeValidation;
 
 import static org.openrewrite.java.Assertions.java;
 
 class MockitoWhenOnStaticToMockStaticTest implements RewriteTest {
 
+    //language=java
+    public static final SourceSpecs CLASS_A = java(
+      """
+        public class A {
+            public static Integer getNumber() {
+                return 42;
+            }
+        }
+        """
```

Comment:
We can still add `SourceSpec::skip` as we don't want it to be considered part of the assertions